projects/step-core/src/lib/components/editable-dropdown-label/editable-dropdown-label.component.ts

```diff
@@ -0,0 +1,48 @@
+import { ChangeDetectorRef, Component, ElementRef, forwardRef, Input, TemplateRef, ViewChild } from '@angular/core';
+import { NG_VALUE_ACCESSOR } from '@angular/forms';
+import { MatSelect } from '@angular/material/select';
+import { EditableComponent } from '../../shared/editable-component';
+
+@Component({
+  selector: 'step-editable-dropdown-label',
+  templateUrl: './editable-dropdown-label.component.html',
+  styleUrls: ['./editable-dropdown-label.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => EditableDropdownLabelComponent),
+      multi: true,
+    },
+  ],
+})
+export class EditableDropdownLabelComponent<T> extends EditableComponent<T> {
+  @Input() items!: T[];
+  @Input() itemTemplate!: TemplateRef<{
+    $implicit: T;
+  }>;
```

Comment:
this means introducing yet another directive, I don't think it would be worth it when we can simply pass it as an input and use native angular bindings; furthermore, the case with the table makes a bit more sense as it kind of provides a common API for building said table, whereas here we simply want to pass in a template, nothing complex going on

Reply:
@neogucky

Reply:
I would be fine with both solutions, as Bojidars Idea is already implemented I would leave it as it is